python_package/examples/tests/pieeg_sever.py

```diff
@@ -0,0 +1,90 @@
+import socket
+import threading
+import time
+import logging
+from brainflow.board_shim import BoardShim, BrainFlowInputParams, BoardIds, BrainFlowError
```

Comment:
remove it also, and for streaming data between nodes and processes, we have streaming board

Reply:
Removed pieeg_server.py as data streaming should be handled by high-level APIs like streaming boards or user implementations.